tests/func/test_cli.py

```diff
@@ -261,3 +263,9 @@ def test_unknown_subcommand_help(capsys):
     captured = capsys.readouterr()
     help_output = captured.out
     assert output == help_output
+
+
+def test_cli_suggestions():
+    with pytest.raises(DvcParserError):
+        _ = parse_args(["psh"])
+        _ = parse_args(["fileas"])
```

Comment:
One more thing here: if `psh` parsing raises `DVCParserError` we will never reach `fileas` part of the code.
What we could do here is:
```suggestion
@pytest.mark.parametrize("cmd", ("psh", "fileas"))
def test_cli_suggestions(cmd):
    with pytest.raises(DvcParserError):
        parse_args([cmd])
```

Reply:
Thanks a lot! I learned something new today. This is much better way to implement this. Let me check and update the commit.